jbmc/regression/jbmc/lambda1/Lambdatest.java

```diff
@@ -76,17 +74,54 @@ public boolean custom(Integer i) {
         return custom.is_ok(i);
     }
 
-    public static void main(String[] args) {
-        // Uses all of the above test functions, to ensure they are loaded under
-        // symex-driven loading:
-
-        Lambdatest lt = new Lambdatest();
-        lt.f(0.0f, 0, 0);
-        lt.i(0);
-        lt.j(0);
-        lt.k(0);
-        lt.l(0);
-        lt.m(0);
+  public static void main(String[] args, int unknown) {
+      Lambdatest lt = new Lambdatest();
+      if(unknown == 0)
+        assert lt.g(1.0f, 2, (x, y) -> x.intValue() + y) == 3; // should succeed
```

Comment:
:pick: I know you didn't name these functions, but if we could maybe rename some of these so less impossible to know what is being tested here `g` -> `executeLambda`